extensions/wikia/PaidAssetDrop/js/PaidAssetDrop.desktop.js

```diff
@@ -0,0 +1,138 @@
+define('ext.wikia.PaidAssetDrop', [
+	'jquery',
+	'wikia.log',
+	'wikia.window'
+], function ($, log, win) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.PaidAssetDrop',
+		articleContentId = '#mw-content-text',
+		assetArticleName = 'MediaWiki:PAD_desktop.html',
+		apiEntryPoint = 'api.php?action=query&prop=revisions&rvlimit=1&rvprop=content&format=json&titles=';
+
+	log('Paid Asset Drop enabled', 'debug', logGroup);
+
+	function isValidDate(dateString) {
+		return !isNaN(Date.parse(dateString));
+	}
+
+	function isConfigValid() {
+		if (typeof win.wgPADConfig === 'undefined') {
+			log('wgPADConfig is undefined', 'debug', logGroup);
+			return false;
+		}
+
+		if (!win.wgPADConfig[0]) {
+			log('No start date set', 'debug', logGroup);
+			return false;
+		}
+
+		if (!win.wgPADConfig[1]) {
+			log('No start date set', 'debug', logGroup);
+			return false;
+		}
+
+		if (!isValidDate(win.wgPADConfig[0])) {
+			log('Start date invalid', 'debug', logGroup);
+			return false;
+		}
+
+		if (!isValidDate(win.wgPADConfig[1])) {
+			log('End date invalid', 'debug', logGroup);
+			return false;
+		}
+
+		return true;
+	}
+
+	function isTodayValid() {
```

Comment:
Yep, now will be more explicit since we can configure in the way the time matters :japanese_ogre: